README.md

```diff
@@ -49,3 +49,7 @@ Files in this repo will be run through the [Black Formatter](https://black.readt
 
 ### Running in Docker
 If you want to run the app in a container, you run the application via `make python-run`. This command will also create a container for the postgres database, and will override the postgres host environment variable to correctly allow the two containers to interact with each other
+
+### API
+
+Navigate to `/server/` and run migrations to create models with `python3 manage.py makemigrations` & `python3 manage.py migrate`, then the server can be started with `python3 manage.py runserver`.
```

Comment:
while we're here we can do some Makefile stuff

Reply:
Yeah, let's chat about what you'd like to see?

Reply:
added two make plans, `api-build` and `api-run`